### PR TITLE
Version bump logger added and logging turned on conditionally in Ktor

### PR DESCRIPTION
**Pull request by scribe**
This turned out to be easier than I thought

### CODE REVIEW

File: src/main/kotlin/com/spectralogic/rioclient/RioClient.kt [hunk 1]
```diff
@@ -45,8 +47,12 @@ class RioClient(
     username: String = "spectra",
     password: String = "spectra",
     private val requestTimeout: Long = 60L * 1000L, // 60 seconds
+    longLivedToken: String? = null,
+    private val verbose: Boolean = false
```
Comment (david-limbach) [inline src/main/kotlin/com/spectralogic/rioclient/RioClient.kt:51]: Do we want to set this in the constructor?
If possible a better solution would be to allow caller to decide which calls are verbose and which aren't.
In the case of RioCruise, we could turn on/off verbose logging while a migration is running.

Comment (david-limbach) [reply src/main/kotlin/com/spectralogic/rioclient/RioClient.kt:51]: How about NOT including verbose in the constructor, and then providing method(s) to toggle it.

Comment (scribe) [reply src/main/kotlin/com/spectralogic/rioclient/RioClient.kt:51]: I think I need to know prior to building the httpClient. By the time they can be toggled I think it is too late. Although maybe I'm not seeing it.

Comment (david-limbach) [reply src/main/kotlin/com/spectralogic/rioclient/RioClient.kt:51]: Nope, we do it in RioBroker.  
LoggingController.setRootLoggingLevel() --> LoggingUtil.requestLoggingLevelChange()

**Review by david-limbach — APPROVED** [review]

**Pull request merged**